i18n/en/docusaurus-plugin-content-docs/current/guides/tech/with-nuxtjs.mdx

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 10
+---
+# Use with NuxtJS
+
+It is possible to implement FSD in a NuxtJS project, but conflicts arise due to differences between the NuxtJS project structure requirements and FSD principles: 
+
+- Initially, NuxtJS offers a project file structure without a `src` folder, i.e. in the root of the project.
+- The file routing is in the `pages` folder, while in FSD this folder is reserved for the flat slice structure.
+
+
+## Adding an alias for the `rc` directory
+
+Add an `alias` object to your config:
+
+```ts
```

Comment:
suggestion: Docusaurus has a way to specify filenames, that can make it clearer where the changes should be made:
```suggestion
```ts title="nuxt.config.ts"
```

Reply:
Then you can also update the code blocks below and replace the comment with a proper title